.github/workflows/build.yml

```diff
@@ -423,7 +431,7 @@ jobs:
           echo "::set-output name=path::${{ env.minizinc_cache_path }}"  # expands variables
```

Comment:
You forgot to bump download-artifact version at lines 424, 504, 584, 684, 717
You have also to add path argument when not yet present to make it work:
```yaml
      - name: Download artifacts
        uses: actions/download-artifact@v3
        with:
          name: wheels
          path: wheels
```